__test__/product-router.test.js

```diff
@@ -0,0 +1,376 @@
+'use strict'
+
+require('./lib/setup.js')
+
+const faker = require('faker')
+const superagent = require('superagent')
+const server = require('../lib/server.js')
+const storeMock = require('./lib/store-mock.js')
+const productMock = require('./lib/product-mock.js')
+
+const apiURL = `${process.env.API_URL}`
+
+describe('/products', () => {
+  beforeAll(server.start)
+  afterAll(server.stop)
+  afterEach(storeMock.remove)
+  afterEach(productMock.remove)
+
```

Comment:
Great job thuroghly testing your routes for 201, 401, 400, and 409 for each of your models!